workspaces/liferay-sample-workspace/language/README.markdown

```diff
@@ -0,0 +1,7 @@
+To add or override translations, place your entries inside the `lang/Language.properties` file. After making changes, run the command `../gradlew buildLang` to generate the default translation files. This command will create copies of all keys and values from your `Language.properties` file into the generated files.
+
+Every time you modify the `Language.properties` file, you should run `../gradlew buildLang` to update the translation files accordingly. Any changes you make directly in the generated translation files will be preserved.
+
+### Important
+
+The translations import will fail if the developer feature flag `LPD-27222` is not enabled.
```

Comment:
As long as we are adding a sample, we can add the feature flag to [portal-ext.properties](https://github.com/drewbrokke/liferay-portal/tree/master/workspaces/liferay-sample-workspace/configs/local/portal-ext.properties), right? What is the Liferay version where this will be active?

Reply:
In 2024.Q3, idk if that's the exact tag. But it is for Q3. I missed that portal-ext.properties file. I still haven't added the flag there yet. 

I was talking with @ethib137 regarding the necessity of this README file. But if I understood correctly, even though this file doesn't exist, the flag will be added to the sample workspace, right @drewbrokke ?

Reply:
Yes I think the flag can be added to the workspace. No harm in adding the flag since it's a no-op if the version is too old (it'll never be read by any active code paths).

Reply:
Just added the feature flag to the workspace